src/core/include/openvino/core/node_output.hpp

```diff
@@ -155,7 +155,7 @@ class OPENVINO_API Output<const Node> {
     /// \return The element type of the output referred to by this output handle.
     OV_NO_DANGLING const element::Type& get_element_type() const;
     /// \return The shape of the output referred to by this output handle.
-    OV_NO_DANGLING const Shape& get_shape() const;
+    OV_NO_DANGLING Shape get_shape() const;
```

Comment:
```suggestion
    Shape get_shape() const;
```
Is `OV_NO_DANGLING ` required when return shape by value (copy)?